server/lib/config/pbench-server.cfg.example

```diff
@@ -12,8 +12,7 @@ install-dir = /opt/pbench-server
 pbench-top-dir = /srv/pbench
 # Backup dir may be remote NFS mounted
 pbench-backup-dir = /mnt/pbench.archive.backup
-# Add role for sync'ing with satellites
-roles = pbench-maintenance, pbench-prep, pbench-results, pbench-backup, pbench-sync-satellites
+roles = pbench-maintenance, pbench-results, pbench-backup
```

Comment:
Can you also mention in the commit message (and perhaps also reflect it in the PR description) that you are also removing the `pbench-sync-satellites` role from the example?

Reply:
But why are you removing the role? Is it not needed any longer?

Reply:
The satellite code is tied to the ssh transport mechanism, and pulls tarballs with no feasible way to track ownership. Peter and I have had some conversations about possible strategies to replace it in the future with something that's authenticated (e.g., replicating user databases, relying exclusively on a shared SSO, etc), but the latest thought is that it probably makes more sense to completely can the "feature". For our purposes, the DMZ pbench will fulfill the original need. If we don't get there we'll have to do something else, but it won't look much like the current satellite polling.

I would have left it alone, but since I was changing the example file's `roles` it made sense to remove the satellite role along with `pbench-prep`.

Reply:
I've modified #2421 after some serious frustration this morning trying to push datasets to my reconstituted server in order to start experimenting with Elasticsearch queries for user/access (#2370). My datasets weren't getting unpacked or indexed. This is because I've managed to completely overlook a flaw in our `PUT` implementation. It only replaces the actual `ssh` operation, *not* `pbench-server-prep-shim-002`. That is, it still relies on `prep-shim` to process the uploaded tarball and MD5, move them to the archive tree, and create the proper state link directories to get the new dataset "into the pipeline".

We really need to close that door completely, so I think we should fix (or "complete") `PUT` so we actually *can* drop `pbench-server-prep-shim-002` entirely. Right now, however, we can't, so I'm going to drop this change.

TBD whether I drop the satellite change, too; we don't want it on 0.72 as it has the same inherent problems as `ssh` upload, compounded by the problems of assigning ownership to satellite datasets. However, if I'm not changing the default `roles` line anyway, it might not really make sense to remove it right now.